Documentation/specs/corehost.md

```diff
@@ -93,6 +93,32 @@ The runtime is located by searching the following paths in order, where `APP_BAS
     * `%LocalAppData%\dotnet\runtime\coreclr\LIBCORECLR`
     * `%ProgramFiles%\dotnet\runtime\coreclr\LIBCORECLR` [2]
 
+
+## CoreHost Servicing
+
+For **RC** milestone, the servicing for the host is tied to the `dotnet/cli` tools i.e., the app developer would have to pull the `corehost.exe` and `hostpolicy.dll` from the appropriate CLI binary drop and replace it in their app or re-publish their app with a serviced CLI.
+
+For **RTM**:
```

Comment:
Can you please call out the motivation for Breadcrumb based servicing as well so that the spec reader has complete context about it?